armbian/build
# Enable odroid-c4 i2c-0

# Description

On Odroid-C4 the i2c-0 is using GPIOX_17(SDA) and GPIOX_18(SCL). 
There were no overlays for meson64 to enable an I2C bus on these GPIO's.
Looking at the meson-g12a-radxa-zero-i2c overlays, a meson-sm1-odroid-c4-i2c overlay is added enable the i2c-2 port which is aliased to i2c-0 on odroid-c4

# How Has This Been Tested?

- Install i2c-tools in device image
- Update armbianEnv.txt by adding overlays=sm1-odroid-c4-i2c-ee-m2-gpiox-17-gpiox-18
- reboot device
- check i2cdetect -l (i2c-0 should be present with Meson I2C adapter driver)
- check i2cdetect -y 0 (any connected device on i2c-0 should be visible)


# Checklist:

_Please delete options that are not relevant._
